test/integration/projects.js

```diff
@@ -104,8 +119,30 @@ describe('test/integration/projects The projects API', () => {
       .catch(helpers.handler);
   });
 
+  it('POST /projects should create a new project with a logo', () => {
+    return agent.post('/projects')
+      .type('form')
+      .field('name', project.name.concat(' with logo'))
+      .field('abbr', project.abbr.concat(' WL'))
+      .field('enterprise_id', project.enterprise_id)
+      .field('zs_id', project.zs_id)
+      .field('locked', project.locked)
+      .attach('logo', fs.createReadStream(file), 'file.jpg')
+      .then(innerRes => {
+        expect(innerRes).to.have.status(201);
+        return agent.get(`/projects/${innerRes.body.id}`);
+      })
+      .then(innerRes => {
+        expect(innerRes).to.have.status(200);
+        expect(innerRes).to.be.json;
+        expect(innerRes.body.name).to.equal(project.name.concat(' with logo'));
```

Comment:
Interesting test!  Is there a way to check if the file path is actually returned?

I'm thinking something like:
```js
expect(innerRes.logoPath).to.include('file.jpg'); 
```

Or something similar?

Reply:
But how do we know that the server succeeded in changing the file name?